packages/api-electron/package.json

```diff
@@ -33,7 +29,11 @@
     "ssf-electron": "./bin/ssf-electron"
   },
   "devDependencies": {
+    "@types/electron-notify": "^0.1.5",
     "concat": "^1.0.3",
+    "containerjs-api-compatibility": "^0.0.6",
+    "containerjs-api-specification": "^0.0.7",
+    "containerjs-api-utility": "^0.0.5",
```

Comment:
is this really a dev dependency?

Reply:
Because it gets rolled-up into build/dist/containerjs-api.js (or containerjs-api-symphony.js), which is the preload file that contains the entire compiled ContainerJS library.

Reply:
Aha - I thought there was a good reason. Thanks for educating me :-)